src/Adapter/MSTest.CoreAdapter/Execution/TypeCache.cs

```diff
@@ -635,9 +636,47 @@ private TestMethodAttribute GetTestMethodAttribute(MethodInfo methodInfo, TestCl
         /// <returns> The <see cref="MethodInfo"/>. </returns>
         private MethodInfo GetMethodInfoForTestMethod(TestMethod testMethod, TestClassInfo testClassInfo)
```

Comment:
Do we need unit tests for the functional changes?

Reply:
The tests are coming in the second installment. I have some tests but couldn’t decide where to put those, I will include those in my next pr.